feat(ttid+ttfd): Add chart showing ttid+ttfd timing for iOS and Android (#9743)
getsentry/sentry-docs: docs/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

_(New in version 8.4.0)_

> Time to initial display (TTID) provides insight into how long it takes to your view controller to launch and draw their first frame, by adding a span for a view controller when it is loaded. The SDK sets the span operation to `ui.load.initial-display` and the span description to the view controllers's name followed by `initial display`, for example, `MainViewController initial display`.

Reviewer: ```suggestion
By adding a span for a view controller when it's loaded, time to initial display (TTID) provides insight into how long it takes for your view controller to launch and draw its first UI frame. The SDK sets the span operation to `ui.load.initial-display` and the span description to the view controllers's name, followed by `initial display` - for example, `MainViewController initial display`.
```

The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment.

The span finishes after the view appeared on the screen.

> The following chart shows how Time to initial display (TTID) and [Time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:
> 

Reviewer: ```suggestion
The following chart shows how time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:

```

_(New in version 8.4.0)_

> Time to full display (TTFD) provides insight into how long it takes your view controller to launch and load all of its content, by adding a span for each launch of a view controller. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers' name followed by `full display`, for example, `MainActivity full display`.

Reviewer: ```suggestion
By adding a span for a view controller when it's loaded, time to full display (TTFD) provides insight into how long it takes your view controller to launch and load all of its content. The SDK sets the span operation to `ui.load.full-display` and the span description to the view controllers' name, followed by `full display` - for example, `MainActivity full display`.
```
